OpenMiChroM/ChromDynamics.py

```diff
@@ -1299,8 +1299,32 @@ def createSimulation(self):
             print(energyInfo, file=f)
             print(f'\nPotential energy per forceGroup:\n {self.getForces()}', file=f)
 
+    def resetSimulation(self, keepforce = False):
+        """ reset the created simulation and  """
+    
+        # delete former Simulation 和 Context
+        if hasattr(self, 'simulation'):
+            del self.simulation
+        if hasattr(self, 'context'):
+            del self.context
+        self.contexted = False  # approve reinitialization
+    
+        # reset forces(avoid iteration) 
+        existing_forces = {type(force).__name__ for force in self.system.getForces()}  # get old force setting
```

Comment:
it may lead to unexpected results if there are multiple forces of the same type. the use of {}  here creates only one entry for each force type